lib/gui/app.js

```diff
@@ -198,6 +198,11 @@ app.config(($provide) => {
   });
 });
 
+app.config(($qProvider) => {
+  // Prevent errors generated by $uibModalInstance.dismiss() unhandled rejections
+  $qProvider.errorOnUnhandledRejections(false);
```

Comment:
This sounds weird. Why shouldn't we error on unhandled rejections? If there was an unhandled rejection, we should have handled it.

If this only related to the modals, can we handle it explicitly instead of disabling it globally here?